data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+			`	"Sent by Consul Aulori, we were," the Kimek says proudly, as if you were supposed to know who that is. "An important task for you, he has in mind."`
+			`	"How far would the visitor go, if to further help our Coalition, it meant?" The Arach asks.`
+			choice
+				`	"I'm not sure I understand what you're getting at."`
+				`	"Obviously I have my limits as for what I would do."`
+			`	"A jump drive, you possess. Far away, you can travel. Bother you, the harsh cages of the Quarg do not," the Saryd explains as all three walk toward your ship.`
+			`	"Few in number, our own jump drives are, and too risky would it be, for one of our own to venture far out," the Kimek adds.`
+			`	"If to help us gain knowledge of the opressors, you wish, lend the Coalition your ship's sensors for a short time, would you?" asks the Arach.`
+			choice
+				`	"Are you asking me to spy on the Quarg?"`
+				`	"Sorry, I don't want to get involved in your conflict with the Quarg."`
+					decline
+			`	"As espionage, a 'malfunction' qualifies not," the Saryd responds.`
+			`	"Merely alter your own ship's sensors, we would. For a time, specially tuned to scan Quarg worlds, they would be," the Kimek clarifies. "No evidence against you, they would have."`
+			choice
+				`	"Alright, which worlds do you need me to land on?"`
+				`	"Sorry, I don't want to get involved in your conflict with the Quarg."`
+					decline
+			`	They ask you to show them your map of the galaxy, so you bring them inside your ship and point out to them all the Quarg planets and stations you have found. They're a bit startled when they first see the entire map, as if unsure about where to send you.`
+			`	You give them a quick rundown of each Quarg world, and after discussing among themselves for a bit, the Saryd speaks. "Interested in the Quarg's mining operations, and the various construction stages of their ringworlds, the consul is."`
+			`	He points to some of the worlds, and you list the places you need to land: <stopovers>.`
+			`	They call for some engineers to come work on your ship's sensors, the whole thing taking just a few minutes. When they're done, the Heliarch agents thank you for your cooperation, and tell you to head for the <destination> when you've collected all the data.`
+				accept
+	to fail
+		"reputation: Quarg" < 0
+	on complete
+		event "heliarch recon break" 10 21
+		"coalition jobs" ++
+		payment 165000
+		conversation
+			`The Heliarch trio are eagerly waiting for you when you return. They are quick to welcome you, urging you to show them your ship's logs so they can see what information you've gathered. They don't seem to pay much mind to data on Grakhord or Forpelog, being much more interested in the ringworlds. As they examine the data, though, the Saryd frowns, while the Kimek brings all his legs closer to his body, and the Arach squints all her eyes - probably their versions of that reaction.`
```

Comment:
Realistically this would be the effort of hours upon hours to really look at the data - unless you're waiting around this should just be a note of what they seem most and least excited to have successfully got data on.

Reply:
They knew what they were looking for and noticed that the data was missing, so I could see them quickly noticing the missing data. Maybe adding a stage of confusion when they first notice the data is gone

Reply:
The implication is that the Quarg just thwarted all attempts to gather data on the rings.
It doesn't take long to analyze the data when there's no data.